lib/http/HttpConnectionsStream.js

```diff
@@ -35,6 +35,7 @@ HttpConnectionsStream.prototype.close = function () {
 
 HttpConnectionsStream.prototype._fetchNextPage = function () {
   var self = this;
+  this.emit('request');
```

Comment:
So, you're not emitting an event when fetching the first page? I'd put the emitter in the HttpFetcher class when on .get